networking/build.gradle

```diff
@@ -21,8 +23,13 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
     // Wolmo
-    compile 'com.github.Wolox:wolmo-core-android:master'
+    compile 'com.github.Wolox:wolmo-core-android:v0.5.0'
 
     // Third party
     compile 'com.squareup.retrofit2:retrofit:2.2.0'
+    compile 'com.squareup.retrofit2:converter-gson:2.2.0'
+    compile 'com.squareup.okhttp3:okhttp-urlconnection:3.7.0'
+    compile 'com.squareup.okhttp3:okhttp:3.7.0'
+    compile 'com.squareup.okhttp3:logging-interceptor:3.7.0'
+    compile 'joda-time:joda-time:2.9.9'
```

Comment:
I'm actually hesitant about enforcing Joda Time for managing time-related stuff because of this:

*Note that from Java SE 8 onwards, users are asked to migrate to java.time (JSR-310) - a core part of the JDK which replaces this project.* Taken from http://www.joda.org/joda-time/.

Since we're getting Java 8 support soon, we can consider dropping it. What are your thought about this?